cpp/src/sort/sort.cu

```diff
@@ -73,22 +76,55 @@ std::unique_ptr<column> sorted_order(table_view input,
         *device_table, *device_table,
         d_column_order.data().get(),
         d_null_precedence.data().get());
-    thrust::sort(rmm::exec_policy(stream)->on(stream),
-                 mutable_indices_view.begin<int32_t>(),
-                 mutable_indices_view.end<int32_t>(), comparator);
-
+    if (stable) {
```

Comment:
Suggesting splitting to 2 files  `sort.cu` and `stable_sort.cu` to reduce compile time. 
real    1m26.625s (sort)
real    2m35.124s (sort and stable_sort)

Reply:
Agreed, I believe this file is the longest to compile file in libcudf, so lets not make it even longer.

Reply:
Although I agree with the sentiment, curiously, my machine fares better:
```
sort
real    0m31.449s
```
anyway, the change is in https://github.com/rapidsai/cudf/pull/4272/commits/ba75615c3c45bda5216c754b9237d7141ca6c2ef

Reply:
Huh, I remember the last time I looked, `sort.cu` took like 5 minutes to compile.